packages/p2p/src/stores/general-store.js

```diff
@@ -502,15 +509,6 @@ export default class GeneralStore extends BaseStore {
                     },
                     [this.updateAdvertiserInfo, response => sendbird_store.handleP2pAdvertiserInfo(response)]
                 ),
-                order_list_subscription: subscribeWS(
-                    {
-                        p2p_order_list: 1,
-                        subscribe: 1,
-                        offset: 0,
-                        limit: this.list_item_limit,
-                    },
-                    [this.setP2pOrderList]
```

Comment:
since you have removed `setP2pOrderList`, where are you calling it now?

Reply:
I can see it was called in app.jsx now 👀

Reply:
Yes I made changes for function.But I forgot to call this method.Now I added

Reply:
how exactly is the method being called? i dont really see it being called anywhere?